development/release.md

```diff
@@ -1,10 +1,10 @@
 # Release Process
 
-The follow process should be followed for a FlowForge release
+The follow process should be followed for a FlowForge release.
 
 ## Setup
 
- - Decide who will be Release Manager for this release. This should be shared across the whole team to prevent it becoming a single point of failure.
+ - Decide who will be Release Manager for this release. For Major/Minor releases this should be shared across the whole team to prevent it becoming a single point of failure. For Fix releases it can be the developer commiting the fix.
```

Comment:
```suggestion
 - Decide who will be Release Manager for this release. For Major/Minor releases this should be shared across the whole team to prevent it becoming a single point of failure. For Fix releases it can be the developer committing the fix.
```